services/server/test/integration/game.js

```diff
@@ -121,6 +121,25 @@ test('GET /game/:id returns 404 if game does not exist', async (t) => {
   t.is(status, StatusCodes.NOT_FOUND);
 });
 
+test('POST /game fails with invalid customURL', async (t) => {
+  const { api } = t.context.app;
+  const userCred = await createUserCred(t);
+  await createUserAndAssert(t, api, userCred);
+
+  const gameRaw = {
+    name: 'first game',
+    description: 'this description',
+    commitSHA: 'published-test-app',
+    githubURL: 'https://github.com/turnbasedgames/urturn',
+    customURL: ',,,--fdsahj',
```

Comment:
Can we have a loop here, and an array of bad values

Reply:
✍️